queries.yaml

```diff
@@ -881,6 +881,73 @@ post_process:
         (kind == 'rail' and zoom < 15) or
         (kind == 'minor_road' and zoom < 14) or
         (kind == 'major_road' and zoom <  7)
+  # drop these "detail" tags to get better merging at zoom < 16
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties:
+        - colour
+        - motor_vehicle
```

Comment:
It looks like `motor_vehicle` = **no** could be used for `minor_road` starting at zoom 12:

https://github.com/tangrams/walkabout-style/blob/gh-pages/walkabout-style.yaml#L2294-L2304